src/main/webapp/views/common/registration.html

```diff
@@ -0,0 +1,51 @@
+<div class="modal fade" id="exampleModalCenter" tabindex="-1" role="dialog" aria-labelledby="exampleModalCenterTitle" aria-hidden="true">
+  <div class="modal-dialog modal-dialog-centered" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h5 class="modal-title" id="exampleModalLongTitle">Account Information</h5>
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+      <div class="modal-body row">   
+        <div id="registration_form_wrapper" class="col-auto text-left form p-4">
+          <form action="/new-vendor" method="POST">
+            <div class="form-row align-items-center justify-content-center">
+              <div class="form-group circle">
+                <img id="profile-picture" src="images/placeholderImage.png" 
+                  class="card-img-top"
+                  alt="images/placeholderImage.png">
```

Comment:
I think a better alt text will be `Placeholder for Profile Pic`, or something that is not a path.

Reply:
For sure!